tests/pre-merge/unit/deployment/test_prediction_converter.py

```diff
@@ -309,9 +309,9 @@ def test_keypoint_to_prediction_converter(self, fxt_label_list_factory):
         [
             (
                 ["1", "2"],
-                ["foo bar", "foo_bar"],
-                ["foo_bar", "foo_bar"],
-                {"label_ids": ["1", "2"], "labels": ["foo_bar", "foo_bar"]},
+                ["foo bar1", "foo_bar2"],
+                ["foo_bar1", "foo_bar2"],
+                {"label_ids": ["1", "2"], "labels": ["foo_bar1", "foo_bar2"]},
```

Comment:
The purpose of this test case was to check an edge case where the label names are very similar and conflict after escaping (both `foo bar` and `foo_bar` become `foo_bar`). The new label names `foo bar1` and `foo_bar2` do not trigger the intended edge case.